src/Symfony/Component/Form/Tests/Extension/Core/Type/CountryTypeTest.php

```diff
@@ -58,6 +58,25 @@ public function testChoiceTranslationLocaleOption()
         $this->assertContainsEquals(new ChoiceView('MY', 'MY', 'Малайзія'), $choices);
     }
 
+    /**
+     * @requires extension intl
```

Comment:
i believe this is only needed with `choice_translation_locale` which triggers a different collation for sorting

Reply:
Aaah good to know.. I've removed it..